### PR TITLE
Fix the build problem. Fix #17

### LINKED ISSUE #17: Error from uglifyJs
I added idle-vue with npm to my vue project and everything was working fine in `run dev` but when i tried to build my project I got the following error:

`ERROR in static/js/vendor.87bf41a35bd9b3a1e693.js from UglifyJs
Unexpected token: punc (() [./~/idle-vue/dist/build.js:678,0][static/js/vendor.87bf41a35bd9b3a1e693.js:12395,9]
`

Did anyone else experience this? I tried changing some settings in my .babelrc but I still get the error.

### PR DESCRIPTION
**Pull request by afonso**

### CODE REVIEW

File: dist/build.js [hunk 1]
```diff
@@ -1,1056 +1,2 @@
...
...und-position:0% 50%\n}\n}\n"],sourceRoot:""}])},function(t,e,n){t.exports=n.p+"static/img/touch.ba3be66.png"},function(t,e,n){function i(t){n(14)}var s=n(1)(n(7),n(12),i,null,null);t.exports=s.exports},function(t,e){t.exports={render:function(){var t=this,e=t.$createElement,n=t._self._c||e;return n("div",{staticClass:"sprite"},[n("canvas",{attrs:{id:t.spriteId,width:t.spriteW,height:t.spriteH}})])},staticRenderFns:[]}},function(t,e,n){t.exports={render:function(){var t=this,e=t.$createElement,i=t._self._c||e;return i("div",{staticClass:"idle-view",class:{isIdle:t.isIdle}},[i("div",{staticClass:"overlay"}),t._v(" "),i("sprite",{ref:"sprite",attrs:{spriteId:"touch",spriteSrc:n(10),spriteW:180,spriteH:215}})],1)},staticRenderFns:[]}},function(t,e,n){var i=n(8);"string"==typeof i&&(i=[[t.i,i,""]]),i.locals&&(t.exports=i.locals);n(2)("b6696284",i,!0)},function(t,e,n){var i=n(9);"string"==typeof i&&(i=[[t.i,i,""]]),i.locals&&(t.exports=i.locals);n(2)("25b5ba8a",i,!0)},function(t,e){t.exports=function(t,e){for(var n=[],i={},s=0;s<e.length;s++){var r=e[s],o=r[0],a=r[1],u=r[2],c=r[3],d={id:t+":"+s,css:a,media:u,sourceMap:c};i[o]?i[o].parts.push(d):n.push(i[o]={id:o,parts:[d]})}return n}}]);
```
Comment (joefresco) [inline dist/build.js]: This should have the same code formatting as the code it replaces above.  That way the exact diff is trivial to determine, and the resulting code is much easier to read.

Comment (afonso) [reply dist/build.js]: This is a build generated code, the fix is the presence of .babelrc and some packages. I can run in a code beautify?

**Comment by joefresco** [comment]
From looking at the code generated, it looks like \n is being interpreted
as a string instead of a literal on the code committed.

I'm not a maintainer, but all the build.js files I've seen are formatted
with typical new lines per statement/structure and indented for
readability.  My own project's build.js is like this and it also is
automatically generated.

**Comment by agnivade** [comment]
This is a simple error which even prevents the project from being built. Should have been caught in the first place if there was a travis setup.

If this PR can be merged, I can send in a PR which adds travis.

**Comment by gabrielstuff** [comment]
Hi @agnivade !
We will review PR today and merge accordingly.

Just after, please feel free to add any CI that you feel intersting.

Thanks !

**Comment by gabrielstuff** [comment]
Thanks !

**Comment by gabrielstuff** [comment]
hey @afonso, sorry it took us so long. 
Would you mind flag as deprecated the duplicate version you published on npm.
Hope we will move faster together.

Thanks !

**Pull request merged**